tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java

```diff
@@ -81,6 +82,43 @@ public Tablet(String deviceId, List<MeasurementSchema> timeseries) {
     this(deviceId, timeseries, DEFAULT_SIZE);
   }
 
+  public void addTimestamp(int rowIndex, long timestamp) {
+    timestamps[rowIndex - 1] = timestamp;
+  }
+
+  public void addValue(MeasurementSchema measurementSchema, int rowIndex, Object value) {
+
+    int indexOfValue = schemas.indexOf(measurementSchema);
```

Comment:
It's not a good way to get the index, measurements maybe very large. You should store the measurementSchema as a map